### ISSUE TITLE
Browser preview creates a million tabs

### ISSUE THREAD

**Issue by santiagobasulto** [auchenberg/vscode-browser-preview#177]
https://user-images.githubusercontent.com/872296/107125823-c3854a00-688a-11eb-8645-a640c11f3bf3.mp4

**Comment by auchenberg** [comment]
@santiagobasulto Very interesting. Does this happen if you only have the browser preview extension installed?

**Comment by santiagobasulto** [comment]
@auchenberg yes, I thought I had hit a weird combination of keys... but if there are others with the same issue, it must not be the case.
